.mailmap

```diff
@@ -1674,3 +1674,5 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
 袁野 (Yuan Ye) <yuanyelele@tutanota.com>
+Peter Stahlecker <peter.stahlecker@gmail.com>   Peter230655 <83544146+Peter@users.noreply.github.com>
+Peter Stahlecker <peter.stahlecker@gmail.com>  Peter Stahlecker <peter.stahlecker@gmail.com
```

Comment:
The two names reported by the mailmap check script are:
```
	Peter Stahlecker <83544146+Peter230655@users.noreply.github.com>
	Peter Stahlecker <peter.stahlecker@gmail.com>
```
What needs to be added is just one line like:
```
Peter Stahlecker <peter.stahlecker@gmail.com> Peter Stahlecker <83544146+Peter230655@users.noreply.github.com>
```
After adding that line you need to run the `bin/mailmap_check.py` script twice. The first run will sort the file into alphabetical order. The second will confirm that the .mailmap file is correct. Then commit and push the changes.

Currently the script shows that your name will be added twice to the AUTHORS file which is not what we want:
```
The following authors will be added to the AUTHORS file at the time of
the next SymPy release.

	Andrew Mosson <amosson@yahoo.com>
	Edward Z. Yang <ezyang@mit.edu>
	Gonzalo Tornaría <tornaria@cmat.edu.uy>
	James Titus <titusjames299@gmail.com>
	Jan Jancar <johny@neuromancer.sk>
	Marek Madejski <marekmadejski@yandex.com>
	Mark van Gelder <mvgmvgmvg@live.com>
	Peter Stahlecker <83544146+Peter0655@users.noreply.github.com>
	Peter Stahlecker <peter.stahlecker@gmail.com>
	Soumendra Ganguly <soumendraganguly@gmail.com>
	Zhuoyuan Li <zy.li@stu.pku.edu.cn>
```

Reply:
So I delete line 1678 and then run bin/mailmap.py twice, correct?
Sorry about this mess. All I suggested to change were two words in a comment of KanesMethod - and it takes 1000 words for peopel to help me.

Reply:
The line added in .mailmap needs to be exactly the one that I showed. The name is not Peter230655 in the commits.

Reply:
> The line added in .mailmap needs to be exactly the one that I showed. The name is not Peter230655 in the commits.

At least im my local ,mailmap it looks like this:
![image](https://github.com/user-attachments/assets/e9360a29-72b8-4e0c-a08a-182fbfdcd303)
is this not correct?

Reply:
is the space between ....com> and Peter230655 important?
Thanks!

Reply:
It should be exactly this:
```
Peter Stahlecker <peter.stahlecker@gmail.com> Peter Stahlecker <83544146+Peter230655@users.noreply.github.com>
```